### DISCUSSION TITLE
LS customs

### DISCUSSION THREAD

**Discussion by vcMoo** [YimMenu/YimMenu#1704] [Ideas]
make a way to save LS Customizations so we dont have to keep redoing the colors or wheels-

**Comment by Yimura** [comment]
Try saving the vehicle as a persist car? That will save the whole vehicle and you can spawn it again using that.

Unless you wish to modify a vehicle you bought then you'll have to go to the mechanic and pay up.

**Reply by LintStalker** [reply]
What does Start LS Custom do?  It seems to bring up the menu, but when I try to purchase something the map comes up instead

**Reply by LintStalker** [reply]
ok, so what's the point of starting it, if you can't do anything with it?

**Reply by LintStalker** [reply]
Yes, but Yimmenu gives you more customizing options then LS Customs, so I guess it's sort of redundant.  But now I understand the point of it.  Thanks!

**Reply by Yimura** [reply]
Start LS Customs used to work, all it did was bring up the normal mechanic and allow you to use it anywhere. R* seems to have broken it in the last update.

**Reply by LintStalker** [reply]
Ok, now I understand.  Thanks!  Personally, I think Yimmenu alone provides better customization, than starting LS customs.

Is it against the rules, to tell people the working around for saving a vehicle, as I have done below?

**Comment by LintStalker** [comment]
> make a way to save LS Customizations so we dont have to keep redoing the colors or wheels-

You can save a vehicle, by doing this:
1) steal a car off the street that can be stored in a 2, 6, or 10 car garage
2) Drive it into your garage, when the cut scene shows you driving in, spawn the vehicle that you want, and you will end up in the garage with that vehicle.  It will have the tracking and insurance.

You can modify the car using Yimmenu while in the garage, but when your done, I think you have to exit the vehicle (so you stay in the garage), to save the changes.  I was playing around with this last night, and I think it worked.

**Reply by mrbxts** [reply]
hi, have u found a way to save custom colours?

**Reply by LintStalker** [reply]
I found that if I exit the vehicle while still in the garage, the color and other modifications stay

**Reply by Tessio** [reply]
dont work for me...
